selinux/defaults/main.yml

```diff
@@ -0,0 +1 @@
+selinux_state: disabled
```

Comment:
I left this for now because it would lead to make several additional changes resp. additions necessary.

Amongst others it reminds me of a general conceptional issue of this project which I would like to discuss carefully first...

Regarding reduced security: If one would for example use the Puppetlabs vagrant-box it would be disabled anyway; Ubuntu also doesn't have SELinux by default. I rolled my own vagrant-box (`th3l0nius/centos-7.0`) were I left unchanged as much as possible.